bottle_server.py

```diff
@@ -71,6 +71,16 @@ def get_exec():
     result = stdout
   return result
 
+# the method sendMetaData returns the MetaData to the Rise4fun website
+# This metadata is found on the file MetadataInfo.json
+# If you want more information about the Metadata. 
+# This can be found on http://www.rise4fun.com/dev.
+@route('/metadata')
+def sendMetaData():
```

Comment:
Is this router function specific for rise4fun? If that is the case, please reflect this on function name, and also check `isRise4Fun` before making response.

Also change the function name to follow google python style: `function_name`.